README.md

```diff
@@ -8,6 +8,21 @@ For details, see the executable documentation in the `vroom/` directory or the
 helpfiles in the `doc/` directory. The helpfiles are also available via
 `:help codefmt` if codefmt is installed (and helptags have been generated).
 
+# Supported File-types
```

Comment:
I think the README is definitely a *must*. I don't really care about the other places. I don't want to have to read code or worse, download the plugin, to know what the plugin supports.